metabase/metabase
# Add instrumentation for "Turn into model" option on saved questions

Closes https://github.com/metabase/metabase/issues/31381

### Description

Adds snowplow schema and event tracking when clicking the `Turn into a model` option when viewing a saved question.


### How to verify
1. Follow [these steps](https://www.notion.so/metabase/Snowplow-integration-5da1f874beda4153b4fccfa6c1e77caa) to setup Snowplow micro locally
2. Open a saved question
3. Go to the options (`...`) and click `Turn into a model`
4. Visit http://localhost:9090/micro/good to see if the event is there

### Checklist
